content/notices/2022-10-27-twilio-dev.md

```diff
@@ -0,0 +1,12 @@
+---
+Title: Test of Twilio Services
+Date: 2022-10-27 10:00
+Category: Dev
```

Comment:
Made a new category that the general public cannot opt into, I can only add people on the database layer and I'm currently the only subscriber.